alfresco-transform-imagemagick/alfresco-transform-imagemagick/src/main/java/org/alfresco/transformer/executors/ImageMagickCommandExecutor.java

```diff
@@ -35,9 +35,19 @@
  */
 public class ImageMagickCommandExecutor extends AbstractCommandExecutor
 {
-    private static final String ROOT = "/usr/lib64/ImageMagick-7.0.7";
-    private static final String DYN = ROOT + "/lib";
-    private static final String EXE = "/usr/bin/convert";
+    private final String ROOT;
+    private final String DYN;
+    private final String EXE;
+
+    public ImageMagickCommandExecutor(String exe, String dyn, String root)
+    {
```

Comment:
Need to add some validation here to check the the values are getting set correctly, and the same goes for all other constructors